<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dqtplotdialog</class>
 <widget class="QDialog" name="dqtplotdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Когерентное излучение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Когерентное излучение</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="m_PB_DrawCoh">
         <property name="text">
          <string>Нарисовать графики</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="m_PB_DrawNonCoh_4">
         <property name="text">
          <string>Сохранить графики</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCustomPlot" name="m_qW_Grayscale" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="QCustomPlot" name="m_qW_Graph" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Некогерентное излучение</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="1">
        <widget class="QPushButton" name="m_PB_DrawNonCohExtra">
         <property name="text">
          <string>Нарисовать графики ФРТ и ЧКХ</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCustomPlot" name="m_qW_Graph_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="m_PB_DrawNonCoh">
         <property name="text">
          <string>Нарисовать графики</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCustomPlot" name="m_qW_Graph_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="m_PB_DrawNonCoh_3">
         <property name="text">
          <string>Сохранить графики</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Задание характеристик</string>
      </attribute>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="geometry">
        <rect>
         <x>179</x>
         <y>130</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Ввести значения</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>291</width>
         <height>111</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
